ompi/communicator/comm.c

```diff
@@ -215,6 +216,11 @@ int ompi_comm_set_nb (ompi_communicator_t **ncomm, ompi_communicator_t *oldcomm,
     if (NULL == newcomm) {
         return OMPI_ERR_OUT_OF_RESOURCE;
     }
+    newcomm->c_name = (char*) malloc (OPAL_MAX_OBJECT_NAME);
```

Comment:
Any reason why you use `OPAL_MAX_OBJECT_NAME` instead of `MPI_MAX_OBJECT_NAME` here? It's currently an alias but if t hat ever changes we'll be in trouble...

Reply:
yes, I thought we are supposed to use the internal constant names. We are using OMPI_SUCCESS and OPAL_SUCCESS internally instead of MPI_SUCCESS for example.